components/widgets/pulsating_dot/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
not sure if we want to keep this index file, I don't know if there is another benefit different than a shorter import statement.

Reply:
0/5 preference on keeping it for shorter import path and conformity with an existing pattern throughout the app (found more than 20 other instances of index files that only re-exported without doing anything else).

Reply:
Since it's already in the folder `pulsating_dot`, renaming it `index.ts` will make it such that there's no need for this extra file.